ui/app/mainui/AppMain.qml

```diff
@@ -1737,14 +1737,25 @@ Item {
             asynchronous: true
             sourceComponent: StatusSearchListPopup {
                 searchBoxPlaceholder: qsTr("Where do you want to go?")
-                model: rootStore.chatSearchModel
+                model: SortFilterProxyModel {
+                    sourceModel: rootStore.chatSearchModel
+
+                    filters: [
+                        FastExpressionFilter {
```

Comment:
Please use `AnyOf` and two `SearchFilter`s. It will be faster and cleaner than `FastExpressionFilter`.